pkg/cli/klothomain.go

```diff
@@ -255,7 +256,7 @@ func (km KlothoMain) run(cmd *cobra.Command, args []string) (err error) {
 	defer analyticsClient.PanicHandler(&err, errHandler)
 
 	updateStream := options.Update.Stream.OrDefault(km.DefaultUpdateStream)
-	analyticsClient.Properties["updateStream"] = updateStream
+	analyticsClient.AppendProperties(map[string]any{"updateStream": updateStream})
```

Comment:
Might be worth adding a singular form of this method so we don't need to make these single-item maps which are cumbersome.

Reply:
Good call.

(the main branch has it the worst of both worlds: it only has this map-based approach, but in a couple places we call that one several times in a row, each time with a one-element map!)